### DISCUSSION TITLE
Unity to React

### DISCUSSION THREAD

**Discussion by jaime-tasca** [jeffreylanters/react-unity-webgl#470] [Help Wanted!] [answered]
Hello guys
First question here about this package
I trying to do a simple thing with a project
I manage to send from React to Unity, but now I'm having trouble with the other way around
The documentantion says that I need to create a React.jslib but I don't understand where should I put it.
Can anyone give a more detail example?
My folder structure is pretty much like this

![image](https://user-images.githubusercontent.com/114996365/206328974-0e2bc70d-ccc3-41e7-8a95-6f8f48b60295.png)

Thanks in advance

**Comment by jeffreylanters** [comment, marked as answer]
Hi! The JSLib goes into your Unity project within the /Assets/Plugins/WebGL/ folder.

**Reply by jaime-tasca** [reply]
Hello Jeffrey
Work as a charm
Thank you very much!
